Logic/src/Users/EditUser.cs

```diff
@@ -23,8 +23,8 @@ public EditUser(ILogger<EditUser> logger, IHttpRequestHandler httpRequestHandler
             _baseApiUrl = Environment.GetEnvironmentVariable("BASE_API_URL");
         }
 
-        [Function("EditUser")]
-        public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post")] HttpRequest req)
+        [Function("UserEdit")]
+        public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "Users")] HttpRequest req)
```

Comment:
Edit should be a patch / put for updates vs overwrites

Reply:
The frontend sends it as a Post Request, I can probably try changing both, if that works out. or I will leave it as it is, because it further sends a patch request to the DB endpoints as expected.

Reply:
Fixed it to be a post request, and it worked as expected!